packages/a/airbrake-browser.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "airbrake-browser",
+  "description": "Official Airbrake notifier for browsers",
+  "keywords": [
+    "exception",
+    "error",
+    "airbrake",
+    "notifier"
+  ],
+  "author": {
+    "name": "Airbrake"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/airbrake/airbrake-js/tree/master/packages/browser",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/airbrake/airbrake-js.git",
+    "directory": "packages/browser"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@airbrake/browser",
+    "fileMap": [
+      {
+        "basePath": "umd",
+        "files": [
+          "*.js?(.map)"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "@(dist|esm)/**/*.@(js|ts|map)"
```

Comment:
I'm not sure if we actually need these -- does anyone else have an opinion here?